Geometry/MuonCommonData/data/v6/mfshield.xml

```diff
@@ -0,0 +1,632 @@
+<?xml version="1.0"?>
```

Comment:
The shield before and after modification:
<img width="1110" alt="screen shot 2017-03-09 at 08 42 19" src="https://cloud.githubusercontent.com/assets/1390682/23740686/fea6655e-04a4-11e7-8f3d-197425b8e518.png">
<img width="1107" alt="screen shot 2017-03-09 at 08 42 59" src="https://cloud.githubusercontent.com/assets/1390682/23740688/fec6d3c0-04a4-11e7-95b9-aa921cc0dd49.png">
Note, there are changes in both Boron Polyethil. and AntiLead shapes - an outer ring of the latter is added:
<img width="1015" alt="screen shot 2017-03-09 at 08 46 15" src="https://cloud.githubusercontent.com/assets/1390682/23740687/fec659b8-04a4-11e7-81c3-dcb1a7146e50.png">